libnetwork/networkdb/networkdb_test.go

```diff
@@ -278,6 +278,8 @@ func TestNetworkDBCRUDTableEntries(t *testing.T) {
 	err = dbs[1].JoinNetwork("network1")
 	assert.NilError(t, err)
 
+	dbs[0].verifyNetworkExistence(t, dbs[1].config.NodeID, "network1", true)
```

Comment:
See commit message, perhaps someone more familiar with the codebase can confirm that we don't need to alter the existing `networkdb` implementation to account for any potential change in `memberlist`?
@thaJeztah ?

Reply:
Not very familiar with this either, but seems to make sense.

Perhaps @arkodg (if he's still around) would know